src/Tutor.Infrastructure/Database/TutorContext.cs

```diff
@@ -117,6 +117,7 @@ private static void ConfigureKcMastery(ModelBuilder modelBuilder)
             modelBuilder.Entity<KnowledgeComponentMastery>().Ignore(kcm => kcm.IsCompleted);
             modelBuilder.Entity<KnowledgeComponentMastery>().Property(kcm => kcm.IsPassed).HasDefaultValue(false);
             modelBuilder.Entity<KnowledgeComponentMastery>().Property(kcm => kcm.IsSatisfied).HasDefaultValue(false);
+            modelBuilder.Entity<KnowledgeComponentMastery>().Property(kcm => kcm.HasActiveSession).HasDefaultValue(false);
```

Comment:
Why do you need to set default values? In C# an unassigned bool is set to false by default. Can we remove these 3 lines of code?

Reply:
Removed in https://github.com/Clean-CaDET/tutor/pull/30/commits/e34f53add5e3f433c8e9e8b6fdd5eb58f6853676.

Reply:
Actually, slight correction, these lines did have a purpose. While an unassigned bool in C# is set to false by default, Npgsql doesn't comply with that when generating scripts, it makes the columns required but without a default value: 
```
CREATE TABLE "KcMastery" (
    "Id" integer GENERATED BY DEFAULT AS IDENTITY,
    "Mastery" double precision NOT NULL,
    "KnowledgeComponentId" integer NULL,
    "LearnerId" integer NOT NULL,
    "HasActiveSession" boolean NOT NULL,
    "IsPassed" boolean NOT NULL,
    "IsSatisfied" boolean NOT NULL,
    CONSTRAINT "PK_KcMastery" PRIMARY KEY ("Id"),
    CONSTRAINT "FK_KcMastery_KnowledgeComponents_KnowledgeComponentId" FOREIGN KEY ("KnowledgeComponentId") REFERENCES "KnowledgeComponents" ("Id"),
    CONSTRAINT "FK_KcMastery_Learners_LearnerId" FOREIGN KEY ("LearnerId") REFERENCES "Learners" ("Id") ON DELETE CASCADE
);
```
This isn't really an issue for normal use, but I generally prefer to have the table generated properly with default values. 

It practically only matters for the seeding scripts for tests, since they have to explicitly insert these fields if there are no default values. I'm fine with either keeping these lines or correcting the seeding script though.